src/components/ViewSet.tsx

```diff
@@ -0,0 +1,14 @@
+interface Set {
```

Comment:
```suggestion
interface ViewSetProps {
```

to not confuse it with `Set` type from the API